docs/manual/devel_documentation.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: rpm.org - Writing Documentation
+---
+# Writing Documentation
+
+There is a lot of 3rd party documentation on RPM and RPM
+packaging. We collect it on the [Documentation
+page](https://rpm.org/documentation.html) on rpm.org. Many of those
+are unfortunately not quite up-to-date.
+
+Get to an current and continuously updated documentation we are
+currently working on an in-tree Reference Manual. While still being less
+complete than some 3rd party documents we can enforce updates for all
+relevant changes and new features.
+
+## Reference Manual
+
+The [Reference
+Manual](https://rpm-software-management.github.io/rpm/manual/) is
+supposed to cover all user facing aspects of RPM but has still a long
+way to go.
+
+The markdown sources of the Reference Manual are found in
+`docs/manual`. They are not build into html or any other format by the
+make file. Instead the `master` branch is turned in to GitHub
+Pages. By sharing the header with the [rpm.org](https://rpm.org) web
+page that links to both it looks like their are one web site.
+
+For working on the Reference Manual it is useful to render the page
+locally. To do that one need to reproduce the relevant infrastructure
+of GitHub Pages - mainly the [Jekyll](https://jekyllrb.com/) static
+website generator. This can be installed from a distribution
+repository (may be named `rubygem-jekyll`) or via the Ruby mechanisms
+e.g. with `gem install jekyll`
+
+GitHub Pages use a number of Jekyll plugins and does more things in the
+background that is not that relevant for our use case. The plugins can
+be found in `docs/_config.yml`. For convenience most are commented
+out. The remaining ones need also to be install (currently
+`jekyll-titles-from-headings` and `jekyll-relative-links`). The former
+is not strictly necessary as we do set titles explicitly. The later
+makes link between the pages work and though navigation much more
+convenient.
+
+Both can also be installed with `gem install
+jekyll-titles-from-headings jekyll-relative-links`.
+
+To render the Reference Manual `jekyll build` can be executed in the
+`docs` dir. It will create a `docs/_site` directory with the rendered
+web page - including html versions of the man pages. Alternativly
+`jekyll serve` will open a web server on localhost.
+
+## Man Pages
+
+The sources of the man pages are found in `docs/man/` in markdown
+format and are converted to troff format as used by the `man` tool by
+the make file in this directory (or building the whole of RPM). They
+are also rendered to HTML together with the Reference Manual.
+
+## Rpm.org
+
+The [rpm.org](https://rpm.org) web page is maintained in a separate
+git project:
+[rpm-web](https://github.com/rpm-software-management/rpm-web). This
+allows storing data like release tar balls that are not acceptable in
+the rpm repository. It is rendered as GitHub Pages and made available
```

Comment:
Does "it is rendered as GitHub Pages" mean that the html & stylesheet look like GitHub Pages? If so, how about:
"It is rendered in the same way as GitHub Pages..."

Reply:
No, it means what is says:  The git project is rendered and made available by GitHub Pages as GitHub Pages.

Reply:
Nah of course. OK then